packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "23.0.0",
+  "version": "24.0.0",
```

Comment:
Right now we are only updating the core version when a release is made (by the release manager). 

You should update the `react-components` package version though.

Reply:
done!